webui/src/appconfig.js

```diff
@@ -11,7 +11,9 @@ window.AppConfig = {
     },
 
     auth: {
-        clientId: '6f9caa6b-b5cf-467d-a326-f25cb0aca8f2'
+        clientId: '6f9caa6b-b5cf-467d-a326-f25cb0aca8f2',
+        //extraQueryParameter: 'prompt=consent',
+        //clientId: '8cc8306e-25d6-4540-b3a8-c858f923de4b',
```

Comment:
Do you really sure, that it's important comments?

Reply:
Yes I am. At least this one //extraQueryParameter: 'prompt=consent'. By this option you can force ADAL to request consent in seconds - no need to google every time how to do that. And the option itself is self explanatory IMHO - no need to write comment for comment. 
The second one, clientId, could be removed, but it is still pretty useful because again, it gives me a way to switch to my app in seconds - and to change my apps permissions. Or, let's share access to the 6f9caa6b... app.